stdlib/LinearAlgebra/test/diagonal.jl

```diff
@@ -455,4 +455,21 @@ end
     @test Transpose(x)*D*x == (Transpose(x)*D)*x == (Transpose(x)*Array(D))*x
 end
 
+@testset "Triangular division by Diagonal #27989" begin
+    for _ in 1:100
```

Comment:
This loop seems unnecessary?

Reply:
You mean that testing with multiple random values is unnecessary, and I should go for a single one?

The major cost is compilation, once compiled, additional random tests are relatively cheap.

Reply:
Right, but we don't do this anywhere else. For testing correctness of the code path it would be more useful with a loop over e.g. `(Float64, ComplexF64)`.

Reply:
Thanks for the suggestion, this actually caught a bug.